Formula/gtk-doc.rb

```diff
@@ -21,41 +23,46 @@ class GtkDoc < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "2e5f324a0c3e8aac532e8946fb00cc578d18184e8fb8d247329b7fbe252054b2"
   end
 
-  depends_on "itstool" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "docbook"
   depends_on "docbook-xsl"
-  depends_on "gettext"
   depends_on "libxml2"
   depends_on "python@3.9"
-  depends_on "source-highlight"
```

Comment:
Wasn't sure if `gettext` was necessary as I didn't see any documentation or build logs on it.
I believe `source-highlight` was dropped as a dependency in 1.30
See: https://gitlab.gnome.org/GNOME/gtk-doc/-/blob/master/NEWS#L63-64
> GTK-Doc is now using python-pygments to do the syntax highlighing. It is not
depending on highligh or source-highligh anymore.